ft-email-auth-provider/src/handlers/create_account.rs

```diff
@@ -71,12 +79,12 @@ fn validate(
             WHERE
                 EXISTS (
                     SELECT 1
-                    FROM json_each ( data -> 'email' -> 'emails' )
+                    FROM json_each ( data -> 'subscription' -> 'subscription' -> 'confirmation-code')
```

Comment:
```suggestion
                    FROM json_each ( data -> 'subscription'  -> 'confirmation-code')
```

Why is there two subscriptions?

Reply:
because: https://github.com/FifthTry/dotcom/pull/40

I'll update this hunk after this PR has merged

Reply:
done in https://github.com/fastn-community/auth/pull/7/commits/8379ff737e82232fc4a8b20b60c154104bc41d81